RELEASE-NOTES.md

```diff
@@ -9,669 +12,344 @@
+|`{cs_version, 10300 }`              |`cs_version = 10300`                   |
+|`{dtrace_support, false}`           |`dtrace_support = off`                 |
 
-- `s3cmd info` fails due to missing `'last-modified` key in return document.
-- `s3cmd get` of 0 byte file fails.
-- Bucket creation fails with status code `415` using the AWS Java SDK.
+`webmachine` section in app.config
 
-## Other Additions
+|      1.5                           |        2.0                            |
+|:-----------------------------------|:--------------------------------------|
+|`{server_name, "Riak CS"}`          |`server_name = Riak CS`                |
 
-- Bucket-level access control lists
-- User records have been modified so that an system-wide unique email
-  address is required to create a user.
-- User creation requests are serialized through `stanchion` to be
-  certain the email address is unique.
-- Bucket creation and deletion requests are serialized through
-  `stanchion` to ensure bucket names are unique in the system.
-- The `stanchion` serialization service is now required to be installed
-  and running for the system to be fully operational.
-- The concept of an administrative user has been added to the system. The credentials of the
-  administrative user must be added to the app.config files for `moss` and `stanchion`.
-- User credentials are now created using a url-safe base64 encoding module.
+#### Items hidden out in 2.0 by default
 
-## Known Issues
+The default value is not changed.
 
-- Object-level access control lists have not yet been implemented.
+| 1.5                                   | 2.0                        |
+|:--------------------------------------|:---------------------------|
+|`{fold_objects_for_list_keys, true}`   |`fold_objects_for_list_keys`|
+|`{n_val_1_get_requests, true}`         |`n_val_1_get_requests`      |
+|`{gc_paginated_indexes, true},`        |`gc.pagenated_indexes`      |
 
-# Riak CS 0.0.3 Release Notes
+#### Items commented out in 2.0 by default
 
-## Bugs Fixed
+Thus all of them are undefined and disabled, except modules. `rewrite_module` and
+`auth_module` will be commented out but the default value won't change from 1.5.
+This is for indicating operators how to change these to OOS API.
 
-- URL decode keys on put so they are represented correctly. This
-  eliminates confusion when objects with spaces in their names are
-  listed and when attempting to access them.
-- Properly handle zero-byte files
-- Reap all processes during file puts
+| 1.5                                   | 2.0                        |
+|:--------------------------------------|:---------------------------|
+|`{rewrite_module, riak_cs_s3_rewrite }`|`rewrite_module`            |
+|`{auth_module, riak_cs_s3_auth },`     |`auth_module`               |
+|`{admin_ip, "127.0.0.1"}`              |`admin.ip`                  |
+|`{admin_port, 8000 }`                  |`admin.port`                |
```

Comment:
Other IP:port settings (listener, riak_host and stanchion_host) are combined but this one is separated.
Is admin.listener better candidate?